File: app/src/main/java/indi/yume/app/passwordbox/model/database/MainDataBase.java
package indi.yume.app.passwordbox.model.database;

import com.raizlabs.android.dbflow.annotation.Database;

/**
 * Created by bush2 on 2016/4/4.
 */
@Database(name = MainDataBase.NAME, version = MainDataBase.VERSION)
public class MainDataBase {
    public static final String NAME = "main_database";

    public static final int VERSION = 1;
}

File: appkotlin/src/main/kotlin/indi/yume/view/appkotlin/database/MainDataBase.kt
package indi.yume.view.appkotlin.database

import com.raizlabs.android.dbflow.annotation.Database

/**
 * Created by bush2 on 2016/4/4.
 */
@Database(name = MainDataBase.NAME, version = MainDataBase.VERSION)
object MainDataBase {
    const val NAME = "main_database"

    const val VERSION = 1
}
